change/@fluentui-react-calendar-compat-b50cbe44-44ed-4b3a-9329-aa9be78d1e75.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: Calendar should memoize default value since it's a new Date() to avoid rerenders.",
```

Comment:
```suggestion
  "comment": "fix(Calendar): Calendar should memoize today's default value since it causes rerenders by creating a new object each time.",
```